src/indexer/services/monitoringService/index.ts

```diff
@@ -0,0 +1,63 @@
+import { CronJob } from "cron"
+import { DateTime } from "luxon"
+import { Transfer, TransferStatus } from "@prisma/client"
+import { PublishCommandInput } from "@aws-sdk/client-sns"
+import { convertMillisecondsToMinutes } from "../../../utils/helpers"
+import TransferRepository from "../../repository/transfer"
+import { logger } from "../../../utils/logger"
+import { NotificationSender } from "./notificationSender"
+
+export enum NotificationType {
+  INCIDENT = "Incident",
+  WARNING = "Warning",
+}
+
+function getNotificationMessage(transfer: Transfer, durationInMins: number, notificationType: NotificationType): PublishCommandInput {
+  if (notificationType == NotificationType.INCIDENT) {
+    return {
+      TopicArn: process.env.TOPIC_ARN,
+      Message: `INCIDENT: Transfer with id ${transfer.id} is pending for ${Math.round(durationInMins)} minutes.`,
+    }
+  } else if (notificationType == NotificationType.WARNING) {
+    return {
+      TopicArn: process.env.TOPIC_ARN,
+      Message: `WARNING: Transfer with id ${transfer.id} is pending for ${Math.round(durationInMins)} minutes.`,
```

Comment:
We probably want to send some tags in the message, so it's easier for our monitoring system to parse the data. Because of that, it's probably better to move the message into .ejs file. Include more information about the transaction into the message (at least txHash instead of transfer.id and domainID)